static/app/views/organizationStats/usageChart/index.tsx

```diff
@@ -321,7 +321,6 @@ function UsageChartBody({
   dataCategory,
   dataTransform,
   chartSeries,
-  chartTooltip,
```

Comment:
are there plans to remove this https://github.com/getsentry/getsentry/blob/master/static/getsentry/gsApp/views/subscriptionPage/reservedUsageChart.tsx#L550

remove chartTooltip from UsageChartProps as well

Reply:
No plans to remove that. The subscription page usage chart seems to be converting units properly right now(?).

Reply:
resolved offline; tldr is that we need it and can't remove chartTooltip